cypress/e2e/right-panel/laser-panel.spec.ts

```diff
@@ -4,218 +4,260 @@ describe('manipulate laser panel', () => {
-//     checkValue(55, 3, 1);
-//     cy.get('#laser-config-dropdown').select('Leather - Engraving');
-//     checkValue(20, 150, 1);
-
-//     cy.get('#laser-config-dropdown').select('Fabric - 3mm Cutting');
-//     checkValue(35, 20, 1);
-//     cy.get('#laser-config-dropdown').select('Fabric - 5mm Cutting');
-//     checkValue(35, 20, 1);
-//     cy.get('#laser-config-dropdown').select('Fabric - Engraving');
-//     checkValue(15, 150, 1);
-
-//     cy.get('#laser-config-dropdown').select('Rubber - Engraving');
-//     checkValue(40, 150, 1);
-//     cy.get('#laser-config-dropdown').select('Glass - Engraving');
-//     checkValue(25, 150, 1);
-//     cy.get('#laser-config-dropdown').select('Metal - Engraving');
-//     checkValue(50, 140, 1);
-//     cy.get('#laser-config-dropdown').select('Metal - Engraving (Diode Laser)');
-//     checkValue(100, 10, 1);
-//   });
-
-//   it('export parameter file', () => {
-//     const cypressDownloadPath = Cypress.env('cypressDownloadPath');
-//     cy.get('#power').clear().type('100').blur();
-//     cy.get('#speed').clear().type('70').blur();
-//     cy.get('#repeat').clear().type('3').blur();
-//     cy.get('div.add-preset-btn').click();
-//     cy.get('.text-input').type('Hi Flux').blur();;
-//     cy.get('.primary').click();
-//     cy.get('[title="Export"]').click();
-//     cy.wait(5000);
-//     cy.readFile(cypressDownloadPath).its('customizedLaserConfigs').its('16').its('name').should('eq', 'Hi Flux');
-//   });
-
-//   it('import parameter file', () => {
-//     cy.get('[title="Import"] > img').click({ force: true });
-//     cy.get('#file-input').attachFile('testfile.json');
-//     cy.contains('Confirm').click();
-//     cy.contains('testFile').should('exist');
-//     cy.get('#laser-config-dropdown').select('testFile');
-//     checkValue(100, 50, 10);
-//   });
-
-//   function checkValue(power, speed, repeat) {
-//     cy.get('#power').should('have.value', power);
-//     cy.get('#speed').should('have.value', speed);
-//     cy.get('#repeat').should('have.value', repeat);
-//   };
-
-//   function openDocumentSettings() {
-//     cy.get('div.menu-btn-container').click();
-//     cy.get(':nth-child(2) > .rc-menu__item').click();
-//     cy.get('.rc-menu > :nth-child(22)').click();
-//   };
+  function checkValue(power, speed, repeat) {
+    cy.get('#power').should('have.value', power);
+    cy.get('#speed').should('have.value', speed);
+    cy.get('#repeat').should('have.value', repeat);
+  };
+
+  function openDocumentSettings() {
+    cy.get('div.menu-btn-container').click();
+    cy.get(':nth-child(2) > .rc-menu__item').click();
+    cy.get('.rc-menu > :nth-child(22)').click();
+  };
+
+  it('set customized List', () => {
+    cy.get('.layer-param-buttons > div.right').click();
+    cy.get('#wood_5mm_cutting').click({ force: true });
+    cy.get('button[class*="ant-btn css-dev-only-do-not-override-ixblex"]').eq(1).click();
```

Comment:
dont use hash classname as selector because it will change each time antd updated